share/wake/lib/core/json.wake

```diff
@@ -154,18 +154,155 @@ export def x // y =
       foldr flatten tail l
   JArray (helper Nil x)
 
+# Compare two JSON values for equality, as a human would judge them.
+# This includes comparing `JObject`s without regard to the internal order of
+# their keys.
 export def x ==/ y = match x y
```

Comment:
I think the mucking around with `==/` is a separate PR that can come after this normalization code has landed. I also remain unconvinced that the re-ordering below is sufficient.